packages/swap-manager/tests/oneinch.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { ChainFamilyMap, ChainInfo } from '@summerfi/sdk/chains'
+import { Address, Percentage, Token, TokenAmount } from '@summerfi/sdk/common'
+import { SwapManagerImpl, SwapData, SwapProvider } from '~swap-manager'
+
+describe('OneInch | SwapManager', () => {
```

Comment:
We shouldn't test external services in unit tests.

Reply:
This is a tricky one. It is not a unit test, rather an integration one, but I don't know what's the approach at the moment in the monorepo for the tests. How to organize those and when to run them. Perhaps we can have a chat to decide if it's not decided yet

Reply:
I've added the indication that this is an Integration test, but we should talk about the approach when you can